global.json

```diff
@@ -1,9 +1,9 @@
 {
   "tools": {
-    "dotnet": "8.0.100-alpha.1.23061.8",
+    "dotnet": "8.0.200",
```

Comment:
[The default `rollForward` policy is `latestPatch`](https://learn.microsoft.com/dotnet/core/tools/global-json#matching-rules), so this will grab latest `8.0.2xx.`